internal/resource/config/authenticationapi/authentication_api_settings_resource.go

```diff
@@ -133,17 +133,17 @@ func (r *authenticationApiSettingsResource) Configure(_ context.Context, req res
 
 }
 
-func readAuthenticationApiSettingsResponse(ctx context.Context, r *client.AuthnApiSettings, state *authenticationApiSettingsResourceModel, expectedValues *authenticationApiSettingsResourceModel) diag.Diagnostics {
-	//TODO different placeholder?
-	state.Id = types.StringValue("id")
+func readAuthenticationApiSettingsResponse(ctx context.Context, r *client.AuthnApiSettings, state *authenticationApiSettingsResourceModel, idStruct *authenticationApiSettingsIdModel) diag.Diagnostics {
+	var diags, valueFromDiags diag.Diagnostics
```

Comment:
You technically don't need two diags variables since only one method returns any diagnostics